src/main/scala/temple/generate/server/go/common/GoCommonGenerator.scala

```diff
@@ -126,6 +126,23 @@ object GoCommonGenerator {
   private[go] def genMethodCall(objectName: String, methodName: String, args: String*): String =
     s"$objectName.${genFunctionCall(methodName, args: _*)}"
 
+  /** Generate method definition */
+  private[go] def genMethod(
+    objectName: String,
+    objectType: String,
+    methodName: String,
+    methodArgs: Seq[String],
+    methodReturn: Option[String],
+    methodBody: String,
+  ): String =
+    mkCode(
+      "func",
+      CodeWrap.parens(objectName, objectType),
```

Comment:
Should objectType deal with the star here instead?

Reply:
Not necessarily, since you could define either `func (f *Foo) method()` or `func (f Foo) method()` if you want reference or value semantics